composer.json

```diff
@@ -27,6 +27,8 @@
     },
     "suggest": {
         "ext-gmp": "GMP is the preferred extension for big integer calculations",
+        "ext-bcmath": "BCmath is the fallback extension for big integer calculations",
+        "phpseclib/bcmath_compat": "BCmath polyfill for servers where neither GMP nor BCmatch is available",
```

Comment:
minor nit: `BCMatch` -> `BCmath` or `bcmath`

Reply:
Oops! Thank you.